dev/Dockerfile_py3

```diff
@@ -0,0 +1,144 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Thomas Beermann <thomas.beermann@cern.ch>, 2017-2020
+# - Vincent Garonne <vgaronne@gmail.com>, 2017-2019
+# - Cedric Serfon <cedric.serfon@cern.ch>, 2017
+# - Frank Berghaus <frank.berghaus@cern.ch>, 2017-2018
+# - Mario Lassnig <mario.lassnig@cern.ch>, 2019
+
+FROM centos:7
+
+RUN yum -y update
+RUN yum -y install epel-release && \
+    yum -y install https://repo.ius.io/ius-release-el7.rpm && \
+    yum clean all && \
+    rm -rf /var/cache/yum
+RUN yum -y install \
+      gcc \
+      httpd \
+      mod_ssl \
+      mod_auth_kerb \
+      python36u-pip \
+      python36-devel \
+      python36-mod_wsgi \
+      python36-m2crypto \
+      gmp-devel \
+      krb5-devel \
+      git \
+      openssl-devel \
+      gridsite \
+      which \
+      libaio \
+      memcached \
+      nmap-ncat \
+      gfal2-all \
+      gfal2-util \
+      gfal2-python3 \
+      fts-rest-cli \
+      xrootd-client \
+      multitail \
+      vim && \
+      yum clean all && \
+      rm -rf /var/cache/yum
+
+RUN git clone https://github.com/rucio/rucio.git /tmp/rucio
+
+ENV RUCIOHOME=/opt/rucio
+RUN mkdir -p $RUCIOHOME
+RUN mkdir -p $RUCIOHOME/etc/multi_vo/tst/etc
+RUN mkdir -p $RUCIOHOME/etc/multi_vo/ts2/etc
+WORKDIR $RUCIOHOME
+RUN mkdir -p \
+      bin \
+      etc \
+      lib/rucio \
+      tools
+
+RUN python3 -m pip install --upgrade pip
+RUN python3 -m pip install --upgrade setuptools
+RUN rm -rf /usr/lib/python3.6/site-packages/ipaddress*
+RUN python3 -m pip install -r /tmp/rucio/etc/pip-requires-client
+RUN python3 -m pip install -r /tmp/rucio/etc/pip-requires
+RUN python3 -m pip install -r /tmp/rucio/etc/pip-requires-test
+RUN python3 -m pip install psycopg2-binary
+RUN ln -s $RUCIOHOME/lib/rucio /usr/local/lib/python3.6/site-packages/rucio
+
+COPY .pep8 .pep8
+COPY .flake8 .flake8
+COPY pylintrc etc/pylintrc
+RUN cp -r /tmp/rucio/etc .
+RUN cp -r /tmp/rucio/tools .
+
+RUN mkdir /var/log/rucio /var/log/rucio/trace
+RUN chmod -R 777 /var/log/rucio
+
+COPY rucio.cfg $RUCIOHOME/etc/rucio.cfg
+COPY rucio_multi_vo_tst.cfg $RUCIOHOME/etc/multi_vo/tst/etc/rucio.cfg
+COPY rucio_multi_vo_ts2.cfg $RUCIOHOME/etc/multi_vo/ts2/etc/rucio.cfg
+COPY alembic.ini $RUCIOHOME/etc/alembic.ini
+COPY aliases-py27.conf $RUCIOHOME/etc/web/aliases-py27.conf
+COPY ui-aliases-py27.conf $RUCIOHOME/etc/web/ui-aliases-py27.conf
+COPY google-cloud-storage-test.json $RUCIOHOME/etc/google-cloud-storage-test.json
```

Comment:
don't you want to prepare an `etc` directory at the level of `Dockerfile` and then use `COPY etc/ $RUCIOHOME/etc/` (or `./etc/` if WORKDIR is $RUCIOHOME)